dask_expr/tests/test_collection.py

```diff
@@ -341,6 +341,58 @@ def test_boolean_operators(func):
     assert_eq(func(pdf), func(df))
 
 
+@pytest.mark.parametrize(
+    "axis", ("columns", 1, "index", 0), ids=lambda axis: f"{axis=}"
+)
+@pytest.mark.parametrize("level", (None, 0, "x"), ids=lambda level: f"{level=}")
+@pytest.mark.parametrize(
+    "fill_value", (None, 1), ids=lambda fill_value: f"{fill_value=}"
+)
+@pytest.mark.parametrize(
+    "op",
+    (
+        "add",
+        "sub",
+        "mul",
+        "div",
+        "divide",
+        "truediv",
+        "floordiv",
+        "mod",
+        "pow",
+        "radd",
+        "rsub",
+        "rmul",
+        "rdiv",
+        "rtruediv",
+        "rfloordiv",
+        "rmod",
+        "rpow",
+    ),
+    ids=lambda op: f"{op=}",
```

Comment:
why do you need ids here?

Reply:
Helped for debugging.. I can remove if they're undesired.

Reply:
Gone https://github.com/dask-contrib/dask-expr/pull/522/commits/82cffb13c6b4873635f5caa647c80f581b38e669